api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -780,6 +796,21 @@ public void endVisit_shouldFailIfValidationErrorsAreFound() {
 		visitService.endVisit(visit, cal.getTime());
 	}
 	
+	@Test
+	public void endVist_shouldSetTheStopDateTimeOfVisit() throws ParseException {
+		executeDataSet(VISITS_ATTRIBUTES_XML);
+		Visit visit = visitService.getVisit(1);
+		
+
+		assertNull(visit.getStopDatetime());
+		SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+		Date date = dateFormat.parse("2019-09-16 00:00:00");
+		visit.setStopDatetime(date);
+		
+		visitService.endVisit(visit, visit.getStopDatetime());
+		
```

Comment:
@gitcliff  can you remove the unnecesary blank lines?

Reply:
sure